plugins/azure-devops-backend/src/api/PullRequestsDashboardProvider.ts

```diff
@@ -35,16 +35,11 @@ export class PullRequestsDashboardProvider {
     private readonly azureDevOpsApi: AzureDevOpsApi,
   ) {}
 
-  public static async create(
```

Comment:
I'd recommend to keep this async in case other async operations need to happen in the future.

Reply:
Reverted this change.